src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class CreateMarketOrderCommand
+    {
+        public string AssetId { get; set; }
+        public AssetPairModel AssetPair { get; set; }
+        public double Volume { get; set; }
+        public OrderAction OrderAction { get; set; }
+        public ClientModel Client { get; set; }        
+        public GlobalSettingsModel GlobalSettings { get; set; }        
```

Comment:
GlobalSettingsModel should be moved out from CreateMarketOrderCommand

Reply:
?